screens/Home.js

```diff
@@ -0,0 +1,196 @@
+import React from 'react';
+import { StyleSheet, Text, View, Button, Alert, Image, TextInput, StackNavigator} from 'react-native';
+import { Font } from 'expo';
+
+export class HomeScreen extends React.Component {
+  state = {
+    fontLoaded: false,
+  };
+  async componentDidMount() {
+    await Font.loadAsync({
+      'double-bubble-shadow': require('./../assets/fonts/Double_Bubble_shadow.otf'),
+      'source-sans-pro': require('./../assets/fonts/SourceSansPro-Regular.ttf'),
+    });
+    this.setState({ fontLoaded: true});
+  }
+  render() {
+    return (
+      <View style={styles.container}>
```

Comment:
Make sure that there are spaces between the curly braces and content: `{ styles.container }`